mcs/class/corlib/System/TermInfoReader.cs

```diff
@@ -45,7 +46,7 @@ namespace System {
 	//
 	// The header is as follows:
 	//
-	//	Magic number (0x1 and 0x1A)
+	//	Magic number (0x11A/0432 or 0x21e/01036 for terminfo2)
```

Comment:
what's `0432` and `01036` ?

Reply:
It's octal notation. I put it there because that's how it's specified in `ncurses` source code.

Reply:
ah ok, I know this with `o` prefix but I just learned that C uses `0` :D